dev/archery/archery/release/core.py

```diff
@@ -468,7 +468,7 @@ def curate(self, minimal=False):
                 parquet.append((self.jira_instance.issue(c.issue), c))
             else:
                 warnings.warn(
-                    f'Issue {c.issue} is not MINOR nor pertains to GH' +
+                    f'Issue {c.issue} does not pertain to GH' +
```

Comment:
Why do we need this?

Reply:
The logic didn't match the message...
This is in an else chain for `if c.issue is None:` -- it doesn't know `if c.title.minor:`

Reply:
I see.